cluster/addons/addon-manager/Makefile

```diff
@@ -15,8 +15,8 @@
 IMAGE=staging-k8s.gcr.io/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v8.6
-KUBECTL_VERSION?=v1.9.3
+VERSION=v8.7
+KUBECTL_VERSION?=v1.10.7
```

Comment:
why not 1.11?

Reply:
Just in case we would need to cherrypick this to 1.10.